JavaScript/JavaScriptSDK/Logging.ts

```diff
@@ -32,10 +32,22 @@
          */
         public static verboseLogging = () => false;
 
-
+        /**
+         * The internal logging queue
+         */
         public static queue = [];
         
         /**
+         * The maximum number of internal events allowed to be sent per page view
+         */
+        private static MAX_ALLOWED_EVENT_LIMIT = 2;
```

Comment:
Will set this in a later commit. Still looking at telem to figure out a good number.